.gitlab/dynamic_test/evaluate.yml

```
@@ -13,3 +13,6 @@ dynamic_test-evaluate-e2e:
     - pip install boto3==1.38.8 # TODO: Remove this before merging, after dda is bumped in test-infra-definitions
     - dda inv -- -e dyntest.evaluate-index --bucket-uri $S3_PERMANENT_ARTIFACTS_URI --commit-sha $CI_COMMIT_SHA --pipeline-id $CI_PIPELINE_ID
   allow_failure: true # Here to evaluate the system, should not impact the CI
+  variables:
+    GIT_STRATEGY: clone # Needs a full clone to avoid issues with the merge-base command using an outdated version of base branch
```

Did we try: (should work fine since https://gitlab.com/gitlab-org/gitlab-runner/-/merge_requests/4288)
```suggestion
    GIT_DEPTH: 0 # full history
```
... otherwise:
```suggestion
    GIT_DEPTH: 42 # set some limits (network bandwidth, overall job execution time)
    GIT_STRATEGY: clone # Needs a full clone to avoid issues with the merge-base command using an outdated version of base branch
```